utils/curl.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018 Bluzelle
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License, version 3,
+// as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#include <include/bluzelle.hpp>
+
+namespace bzn::utils::curl
+{
+    std::string perform_curl_request(const char* url, const std::string& request);
```

Comment:
why const char*?  Please change to std::string&

Reply:
I was following the style of the curl library.. unnecessarily. Changed.